CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+# dbt_hubspot v0.17.1
+[PR #140](https://github.com/fivetran/dbt_hubspot/pull/140) includes the following updates:
+
+## Under the Hood
```

Comment:
I think it would be appropriate to categorize this as a Bug Fix instead of an under the hood change
```suggestion
## Bug Fixes
```